lib/modules/manager/npm/post-update/types.ts

```diff
@@ -32,6 +32,7 @@ export interface GenerateLockFileResult {
 
 export interface PnpmLockFile {
   lockfileVersion?: number;
```

Comment:
```suggestion
   lockfileVersion?: number | string;
```
is that really optional?